Sources/Purchasing/Purchases.swift

```diff
@@ -1958,6 +1996,16 @@ internal extension Purchases {
                                                                           completion: completion)
     }
 
+    // Used for testing
+    var networkTimeoutSeconds: Int {
+        return self.backend.networkTimeoutSeconds
+    }
+
+    // Used for testing
+    var storeKitTimeoutSeconds: DispatchTimeInterval {
+        return self.productsManager.requestTimeout
+    }
+
```

Comment:
Maybe we should have these in the tests instead?

Reply:
or as a separate file?

Reply:
Oh I remember why I didn't do that- I'm reaching into a private variable that isn't exposed to the test targets.